packages/create-discord-bot/src/index.ts

```diff
@@ -0,0 +1,39 @@
+import { program } from 'commander';
+import prompts from 'prompts';
+import { createDiscordBot } from './create-discord-bot.js';
+
+program
+	.description('Create a basic discord.js bot.')
+	.option('--directory', 'The directory where this will be created.')
+	.option('--typescript', 'Whether to use the TypeScript template.')
+	.option('--javascript', 'Whether to use the JavaScript template.')
+	.parse();
+
+let { typescript, javascript, directory } = program.opts();
+
+if (!directory) {
+	directory = (
+		await prompts({
+			type: 'text',
+			name: 'directory',
+			initial: 'my-bot',
+			message: 'What is the name of the directory you want to create this project in?',
+		})
+	).directory;
+}
+
+if (typescript === undefined && javascript === undefined) {
+	const { useTypescript } = await prompts({
```

Comment:
as I rember from my case
exiting program here will make it undefined
so you should check if it isnt